source/III.VC.CLEO/CustomOpcodes.cpp

```diff
@@ -2014,17 +2014,16 @@ eOpcodeResult CustomOpcodes::OPCODE_0ADA(CScript *script)
 eOpcodeResult CustomOpcodes::OPCODE_0ADB(CScript *script)
 {
 	script->Collect(2);
-	unsigned mi = game.Scripts.Params[0].nVar;
+	auto modelId = game.Scripts.Params[0].nVar;
 	char *result = game.Scripts.Params[1].cVar;
+
 #if CLEO_VC
-	char *gxt = (char*)((game.Misc.stVehicleModelInfo + 0x32) + ((mi - 130) * 0x174));
+	char *gxt = (char*)((game.Misc.stVehicleModelInfo + 0x32) + ((modelId - 130) * 0x174));
 #else
-	char *gxt = (char*)((game.Misc.stVehicleModelInfo + 0x36) + ((mi - 90) * 0x1F8));
+	char *gxt = (char*)((game.Misc.stVehicleModelInfo + 0x36) + ((modelId - 90) * 0x1F8));
 #endif
-	
-	wchar_t *text = CustomText::GetText(game.Text.CText, 0, gxt);
-	wcstombs(result, text, wcslen(text));
-	result[wcslen(text)] = '\0';
+
+	strcpy(result, gxt);
```

Comment:
is the result an address stored in the variable, or the variable itself?

Reply:
Result is pointer to text buffer.

Reply:
I suppose it might not be a variable? Can I provide a static address?

0ADB: index 123 address 0x0ABCDEF

Reply:
In theory somebody can provide this way some known address within game's exe and write label hare.

Maybe better solution will be to use switch on param type like I did in other branch and accept only numbers provided via variables:
```case eParamType::PARAM_TYPE_LVAR:
case eParamType::PARAM_TYPE_GVAR:```

Reply:
The reason I'm asking this, is that currently SBL documents this opcode with the second parameter being a variable

https://library.sannybuilder.com/#/vc/CLEO/0ADB?p=1&v=1

if we can use a static integer here, then we should change the documentation

Reply:
Right, Theoretically one can create working code with hard coded memory address, but script like that will only works with certain game version. I don't think we should encourage that style.

I think documentation generally is bit ambiguous about "string" as parameters/return values. Original opcodes support only regular short/long string variables, meanwhile custom opcodes allows also pointers. It should be somehow indicated in documentation.